cmd/horcrux/cmd/cosigner.go

```diff
@@ -0,0 +1,180 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+	"net"
+	"os"
+	"path"
+	"sync"
+	"time"
+
+	"github.com/jackzampolin/horcrux/internal/signer"
```

Comment:
NIT: we should move the directory from `internal/signer` to `signer` and replace the import path throughout the code.